.nais/naiserator-dev.yaml

```diff
@@ -43,6 +43,12 @@ spec:
     application:
       enabled: true
       tenant: trygdeetaten.no
+  gcp:
+    sqlInstances:
+      - type: POSTGRES_12
```

Comment:
Kanskje vurdere versjon 13 hvis det er slik at denne skal lages nå?

Reply:
Ja hvis det er kommet, så tar vi jo gjerne det

Reply:
Bra tips det! Har allerede opprettet DB i dev, men kan se om jeg kan få slettet den.

Reply:
Ser forøvrig at gir lite eller ikke noe ekstrarbeid om man ikke er på nyeste versjon, ref:
`Cloud SQL will also provide tools to minimize upgrade disruption, as needed. Any instance that has not moved to the new major version at the end of the 12-month period will be automatically upgraded.`